### PR TITLE
Added some fixes per jonno's request

### PR DESCRIPTION
**Pull request by eddieirwins**

### CODE REVIEW

File: addevents.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+==========
+
+AddEvent.CreateLink
+-------------------
+A function to create the "Click to Add Calendar" new feature. This function has a lot of parameters, some are optional. 
+You can have a look at them here: https://www.addevent.com/documentation/direct-url-method
+
+Overloads
+~~~~~~~~~
+1. String link = AddEvent.CreateLink(String client, String startDate, Int duration, String title, String description, String location, String service, String calname);
```
Comment (jonathonc) [inline addevents.rst]: Can you change client to clientId

File: html.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
+HTML
+====
+
+Html.MailTo
+------------
+A function to create the email mailto link, includes encoding to handle things like "&"
+
+Overloads 
+~~~~~~~~~
+1. **Html.MailTo(String email, String subject, String body)**
```
Comment (jonathonc) [inline html.rst]: Can you add a return  type to the function?

File: index.rst [hunk 3]
```diff
@@ -36,3 +36,5 @@ Table of Contents
    tryconvert.rst
    uriaddress.rst
    zip.rst
+   addevents.rst
```
Comment (jonathonc) [inline index.rst]: can you put in alphabetical order

File: html.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Sample:: 
+
+    String email = 'jonathonc@irwinsolutions.com'
+    String subject = 'Does 1 + 10 / 2 = 1000 & more maths...?'
+    String body = 'Hello\nThis is a new line\nDoes 1 + 10 / 2 = 1000 & more maths...?'
+
```
Comment (jonathonc) [inline html.rst]: There is no actually usage of the function?

File: uriaddress.rst [hunk 5]
```diff
@@ -45,4 +45,30 @@ Overloads
...
+  UriAddress google = UriAddress.TryParse("https://www.google.com");
+
+
+UriAddress.EncodeMailToPart
+---------------------------
+Url encodes text for use within mail to query strings
+
+Overloads
+~~~~~~~~~
+1. UriAddress.EncodeMailToPart(String str)
```
Comment (jonathonc) [inline uriaddress.rst]: there is no return type

File: uriaddress.rst [hunk 6]
```diff
@@ -45,4 +45,30 @@ Overloads
...
+
+Sample:: 
+
+  UriAddress.EncodeMailToPart('Does 1 + 10 / 2 = 1000 & more maths...?');
```
Comment (jonathonc) [inline uriaddress.rst:61]: can you include the output?

File: uriaddress.rst [hunk 7]
```diff
@@ -45,4 +45,30 @@ Overloads
...
+--------------------------------
+Url encodes text for use within query strings
+
+Overloads
+~~~~~~~~~
+1. UriAddress.EncodeQueryStringPart(String str)
+
+Sample::
+
+  String urlPart = UriAddress.EncodeQueryStringPart('Does 1 + 10 / 2 = 1000 & more maths...?');
```
Comment (jonathonc) [inline uriaddress.rst:74]: can you include the output?

File: uriaddress.rst [hunk 8]
```diff
@@ -45,4 +45,30 @@ Overloads
...
+  UriAddress.EncodeMailToPart('Does 1 + 10 / 2 = 1000 & more maths...?');
+
+
+UriAddress.EncodeQueryStringPart
+--------------------------------
+Url encodes text for use within query strings
+
+Overloads
+~~~~~~~~~
+1. UriAddress.EncodeQueryStringPart(String str)
```
Comment (jonathonc) [inline uriaddress.rst]: can you include the return type?

**Review by jonathonc — COMMENTED** [review]
@eddieirwins are you able to make the changes according to the comments, let me know if you need me to walk through it with you?

**Comment by eddieirwins** [comment]
Hi Jonno, I believe I have amended accordingly. Could you please review and let me know? Thanks!

File: addevents.rst (hunk 1 above)
Comment (eddieirwins) [reply addevents.rst]: Done

**Pull request merged**